src/FormBuilder/Routes/RegisterFormBuilderPageRoute.php

```diff
@@ -111,6 +112,11 @@ public function renderPage()
                 'isInstalled' => defined('GIVE_RECURRING_VERSION'),
             ],
             'gatewaySettingsUrl' => admin_url('edit.php?post_type=give_forms&page=give-settings&tab=gateways'),
+            'templateTags' => array_values(give()->email_tags->get_tags()),
+            'emailNotifications' => array_map(static function ($notification) {
+                return EmailNotificationData::fromLegacyNotification($notification);
+            }, apply_filters('give_email_notification_options_metabox_fields', array(), $donationFormId)),
+            'emailPreviewURL' => rest_url('givewp/next-gen/email-preview'),
```

Comment:
This object is starting to get big enough where we can move into the viewModel so we can write a unit test for it's shape.

`$formBuilderViewModel->getformBuilderData()`